specification/securityinsights/data-plane/Microsoft.SecurityInsights/preview/2024-02-01-preview/ThreatIntelligence.json

```
@@ -0,0 +1,189 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Sentinel Upload API",
+    "description": "Specification for Microsoft Sentinel Upload API.",
+    "version": "2024-02-01-preview"
+  },
+  "host": "api.ti.sentinel.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/workspaces/{workspaceId}/threatintelligencestixobjects:upload": {
+      "post": {
+        "x-ms-examples": {
+          "Upload Stix Objects": {
+            "$ref": "./examples/threatintelligence/UploadStixObjects.json"
+          }
+        },
+        "summary": "Upload Stix objects to the workspace",
+        "description": "Upload an array of Stix objects to the workspace specified",
+        "operationId": "ThreatIntelligenceStixObjects_Upload",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "workspaceId",
+            "description": "The workspace ID for the Stix objects to be uploaded.",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "maxLength": 36,
+            "pattern": "^[{]?[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}[}]?$"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Body of the request with an array of Stix objects and other properties",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/UploadStixObjectsRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The API returns 200 when at least one Stix object is successfully validated and published.",
+            "schema": {
+              "$ref": "#/definitions/UploadStixObjectsResponse"
+            }
+          },
+          "default": {
+            "description": "Upload Stix Objects API service error response.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "security": [
+          {
+            "azure_auth": [
+              "user_impersonation"
+            ]
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "JsonPropertyBag": {
+      "description": "An object that stores a list of JSON properties from a given JObject.",
+      "type": "object"
```

You should be explicit here that there are unnamed properties in this object.
```suggestion
      "type": "object",
      "additionalProperties": {}
```

But I should also say that this will give a very bad user experience. At the very least you should indicate in the description where users can find definitions for the objects that can be sent.